src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread.scss

```diff
@@ -1,14 +1,25 @@
 @import '../../../common/variables';
 
-.bcs-ActivityThread-toggle {
-    margin-left: $sidebarActivityFeedSpacingHorizontal;
-    color: $bdl-box-blue;
-    font-weight: bold;
+.bcs-ActivityThread {
+    .bcs-ActivityThread-toggle {
+        color: $bdl-box-blue;
 
-    &:hover,
-    &:active {
-        margin-left: $sidebarActivityFeedSpacingHorizontal;
-        font-weight: bold;
+        &,
+        &:hover,
+        &:active {
+            margin-top: -$bdl-grid-unit * 2; // to decreased spacing with parent comment
+            margin-left: $sidebarActivityFeedSpacingHorizontal;
+            font-weight: bold;
+        }
+    }
+
+    .bcs-ActivityCard {
+        margin: $sidebarActivityFeedSpacingVertical $sidebarActivityFeedSpacingHorizontal;
+        padding: 0;
+    }
+
+    button:focus-visible {
+        outline: -webkit-focus-ring-color auto 1px;
```

Comment:
I don't think I've seen this before. Is this a standard visual treatment? Can we reuse the same treatment as hover and active, instead?

Reply:
is an equivalent needed for Firefox?

Reply:
It's about issue Dawid mentioned: https://github.com/box/box-ui-elements/pull/3110#discussion_r1004231413